inc/class-cachify.php

```diff
@@ -825,21 +799,18 @@ public static function edit_comment( $id ) {
 	 * @since   0.1.0
 	 * @change  2.1.2
 	 *
-	 * @param   mixed $approved  Comment status.
-	 * @param   array $comment   Array of properties.
-	 * @return  mixed            Comment status.
+	 * @param	integer			$id			Comment ID
+	 * @param	integer|string	$approved	Comment status
 	 */
-	public static function pre_comment( $approved, $comment ) {
+	public static function new_comment( $id, $approved ) {
```

Comment:
There would be an easy way to keep the existing method without breaking the API I guess.

Reply:
https://developer.wordpress.org/reference/hooks/comment_post/
comment_post was introduced in 1.2

Reply:
resolved in feab290d58cb32083c8fd2ae105930cab2dac7d3